src/ontology/cl.Makefile

```diff
@@ -337,13 +337,25 @@ DEPLOY_GH=true
 
 .PHONY: cl
 cl:
-	$(MAKE) prepare_release IMP=false PAT=false 
-	$(MAKE) release-diff
+	$(MAKE) prepare_release IMP=false PAT=false MIR=false
+	$(MAKE) release-base-diff
+	$(MAKE) prepare_content_summary
 	if [ $(DEPLOY_GH) = true ]; then 	$(MAKE) deploy_release GHVERSION="v$(TODAY)"; fi
 
-.PHONY: release-diff
-release-diff:
-	$(ROBOT) diff --labels True -f markdown --left-iri http://purl.obolibrary.org/obo/cl.owl --right ../../cl.owl --output diffs/$(ONT)-diff.md
+.PHONY: release-base-diff
+release-base-diff:
+	$(ROBOT) diff --labels True -f markdown --left-iri http://purl.obolibrary.org/obo/cl/cl-base.owl --right ../../cl-base.owl --output reports/$(ONT)-base-diff.md
```

Comment:
Why not use 

```
--left $(TMPDIR)/current-base-release.obo
```

(and add as dependency) to avoid some redundant download time?

Reply:
You're right. I didn't update this rule after I added the OAK diff command. I'll change it.

Reply:
done.